launch/tier4_system_launch/config/system_monitor/voltage_monitor.param.yaml

```diff
@@ -0,0 +1,4 @@
+/**:
+  ros__parameters:
```

Comment:
You should add `cmos_battery_voltage`. (but it should be `cmos_battery_label`?)

Reply:
I think so.
Changed the label name to cmos_battery_label.